Latex/vorlage/vorlage.tex

```diff
@@ -475,3 +464,23 @@
   \caption{#2}
   \label{fig:#3}
   \end{figure}}
+\newcommand{\logisch}[1]{$``#1``$}
+\newcommand{\vglink}[2]{\footnote{\hspace{0.5em}vgl.~\href{#1}{#1}~(#2)}}
+\newcommand{\python}[1]{\mintinline{python}{#1}}
+
+% Referenzierung
+% Kapitel
+\newcommand{\fullref}[1]{(\emph{\hyperref[{#1}]{siehe Kapitel \ref{#1} - \nameref{#1}}})}
+\newcommand{\literef}[1]{\emph{\hyperref[{#1}]{Kapitel \ref{#1} - \nameref{#1}}}}
+% Anhänge
+\newcommand{\litearef}[1]{\emph{\hyperref[{#1}]{Anhang \ref{#1}}}}
+\newcommand{\aref}[1]{\litearef{#1}} %Kompatibilität
+\newcommand{\fullaref}[1]{(siehe \emph{\hyperref[{#1}]{Anhang \ref{#1}}})}
+% Abbildungen
+\newcommand{\litebref}[1]{\emph{\hyperref[{fig:#1}]{Abbildung \ref{fig:#1} - \nameref{fig:#1}}}}
+\newcommand{\bref}[1]{\litebref{#1}} %Kompatibilität
+\newcommand{\fullbref}[1]{\emph{(siehe \hyperref[{fig:#1}]{Abbildung \ref{fig:#1} - \nameref{fig:#1}}})}
+% Tabellen
+\newcommand{\litetref}[1]{\emph{\hyperref[{#1}]{Tabelle \ref{#1} - \nameref{#1}}}}
+\newcommand{\tref}[1]{\litetref{#1}} %Einheitlichkeit
+\newcommand{\fulltref}[1]{\emph{(siehe \hyperref[{#1}]{Tabelle \ref{#1} - \nameref{#1}}})}
```

Comment:
vorschlag: alphabetisch sortieren?

Reply:
ne, alphabetisch ist blöd, weil die befehle ja selbst ausgedacht sind, andere leute denken da anders

Reply:
abgelehnt xD das is quasi grad nach kommandos sortiert. erst Kapitel weil am häufigsten, dann aref, bref, tref (alphabetisch)...
außer das findest du sehr bleyde?

Reply:
jetzt erst gemerkt, wie gut strukturiert die (vor allem bei a-, b- und tref) da stehen. passt von mir aus